### PR TITLE
SAK-506 create changelog parameters from array fix

### PR DESCRIPTION
**Pull request by rafalkanski**
@filipgolonka @wsromek @arkadiuszmalina @bartosz-kahl

### CODE REVIEW

**Comment by wsromek** [comment]
👍

**Comment by barkahl** [comment]
👍

**Comment by arkadiuszmalina** [comment]
👍

File: src/Snt/Capi/Repository/Article/FindByChangelogParameters.php [hunk 1]
```diff
@@ -47,12 +49,21 @@ public static function createForPublicationIdFromArray($publicationId, array $pa
...
         $self->publicationId = $publicationId;
 
+        if (isset($parameters['since'], $parameters['until'])) {
+            try {
+                $self->timeRange = new TimeRangeParameter(
+                    new DateTime($parameters['since']),
+                    new DateTime($parameters['until'])
+                );
+            } catch (Exception $exception) {
+            }
```
Comment (filipgolonka) [inline src/Snt/Capi/Repository/Article/FindByChangelogParameters.php]: I'm not sure if it is correct approach

Comment (rafalkanski) [inline src/Snt/Capi/Repository/Article/FindByChangelogParameters.php]: what do you suggest?

Comment (filipgolonka) [inline src/Snt/Capi/Repository/Article/FindByChangelogParameters.php]: do not catch exception here, SDK user should know when `$parameters` are incorrect. IMHO we can leave it just without try..catch clause

Comment (rafalkanski) [inline src/Snt/Capi/Repository/Article/FindByChangelogParameters.php]: 👍

**Comment by rafalkanski** [comment]
@filipgolonka review

**Comment by filipgolonka** [comment]
:+1:

**Pull request merged**